src/metabase/driver.clj

```diff
@@ -708,18 +708,15 @@
   query)
 
 ;; TODO - we should just have some sort of `core.async` channel to handle DB update notifications instead
-;;
-;; TODO -- shouldn't this be called `notify-database-updated!`, since the expectation is that it is done for side
```

Comment:
I've updated the `notify-database-updated!` because of this comment. If it is not desired please let me know and I'll revert.

Reply:
I would rather see this in a seperate pr, this would also need to update the driver changelog/docs/release notes for third party drivers. I'm not 100% sure of this process.

Reply:
We don't rename driver methods like that. I think the standard procedure is deprecating the undesired thing (the old name in this case) but keeping it around for a few versions (not fewer than three, I guess). And yes, the change has to be documented in `docs/developers-guide/driver-changelog.md`.

Reply:
Thanks for explanation. Change was removed.